scss/titles.scss

```diff
@@ -0,0 +1,14 @@
+.main-title {
```

Comment:
check indentation in this whole file. 
Remember that indentation should be `2 spaces or 1 tab`